controllers/provisioners/eks/eks.go

```diff
@@ -114,13 +124,13 @@ func (ctx *EksInstanceGroupContext) GetInstanceGroup() *v1alpha1.InstanceGroup {
 
 func (ctx *EksInstanceGroupContext) GetOsFamily() string {
 	var (
-		instanceGroup  = ctx.GetInstanceGroup()
-		annotations = instanceGroup.GetAnnotations()
+		instanceGroup = ctx.GetInstanceGroup()
+		annotations   = instanceGroup.GetAnnotations()
 	)
 	if _, exists := annotations[OsFamilyAnnotation]; exists {
 		return annotations[OsFamilyAnnotation]
 	}
-	return "default"  // return "" is will also be fine here
+	return OsFamilyAmazonLinux2 // return "" is will also be fine here
```

Comment:
I think we can also remove the comment 😅  - I think it was a review comment that went in with the suggestion

Reply:
lol - yep